style-guide/styles/wds-components/_banner-notification.scss

```diff
@@ -0,0 +1,52 @@
+.wds-banner-notification-container {
```

Comment:
This is not a separate component - it's tied to the `banner-notification` and `banner-notification` cannot be used without the container. I suggest at leat renaming it to `banner-notification__container`.